analysis/source/simulation/designs/baseline.csv

```diff
@@ -1,2 +1,2 @@
-sample_size, 1000
-no_faces, 6
\ No newline at end of file
+sample_size,500
```

Comment:
@cindyjulianelu This looks right to me, but note that the corresponding output has not changed: The histogram still appears to be based on 1000 simulations (the file is unchanged compared to the master branch). Similarly the sample size that is saved [here](https://github.com/SimonFreyaldenhoven/example_template/blob/23-update-sample-size/analysis/output/simulation/baseline/sample_size.txt) still says 1000 and is unchanged from the master branch.